src/app/components/HoldConfirmation/HoldConfirmation.jsx

```diff
@@ -128,32 +121,38 @@ class HoldConfirmation extends React.Component {
 
         <div className="content-wrapper">
           <div className="item-header">
-            <h1>Research item request confirmation</h1>
+            <h1>Request Confirmation</h1>
+            <h2>Submission Received</h2>
           </div>
 
           <div className="item-summary row">
             <div className="details two-third">
-              <h2>Item request details</h2>
-              <ul className="generic-list">
-                <li>Item: <Link to={`/bib/${id}`}>{title}</Link></li>
-                {shelfMarkInfo}
-                { /* <li>Ready for use by <strong>approximately {dateDisplay}, 9:00am</strong> at the location below</li> */ }
-                <li><strong>You will receive an email notification</strong> when the item is ready for use at the location below</li>
-                { /* <li>Book will be held until {dateDisplayEnd}, 5:00pm</li> */ }
-              </ul>
-            </div>
-            <div className="actions third">
-              <h2>Available actions</h2>
-              <ul className="generic-list">
-                <li>Visit your patron account page to view the status of this item hold</li>
-                { /* <li>You may <a href="#cancel">cancel</a> this item hold at any time</li> */ }
-              </ul>
-            </div>
-          </div>
-
-          <div className="map-container">
-            <div className="third">
-              {this.renderLocationInfo(deliveryLocation)}
+              <h2>Item Information</h2>
+              <p>We've received your request for <Link to={`/bib/${id}`}>{title}</Link></p>
+              <p>
+                Please check your library account for updates. The item will be listed as 
+                Ready under your Holds tab when it is available. You will also recieve an email 
+                confirmation after your item has arrived.
+              </p>
+              <p>
+                Your item will be delivered to: {this.renderLocationInfo(deliveryLocation)}
```

Comment:
Sometimes there are no delivery locations or EDD request, so we don't want to display "Your item will be delivered to: " without any details.